lib/os/thread_posix.cpp

```diff
@@ -74,7 +81,7 @@ namespace pstore {
             // - three argument form supported by NetBSD.
             // - the slightly differently named form used in FreeBSD.
             int err = 0;
-#    ifdef USE_FALLBACK
+#    if USE_FALLBACK
```

Comment:
what is the benefit of changing from `#ifdef` to `#if`?

Reply:
Because the change sets USE_FALLBACK to 1 or 0. In both cases it’s defined but I need to treat it as a boolean.